api/v1alpha1/volumereplicationgroup_types.go

```diff
@@ -152,17 +159,46 @@ const (
 
 const KubeObjectProtectionCaptureIntervalDefault time.Duration = 5 * time.Minute
 
+type ResourceCaptureInstance struct {
+	Name string `json:"name,omitempty"`
+
+	//+optional
+	IncludeClusterScopedResources bool `json:"includeClusterScopedResources,omitempty"`
```

Comment:
I suggest the `Scoped` be dropped and it be named `IncludeClusterResources` to align with velero's `Backup` and `Restore` APIs